### PR TITLE
chore(weave): Expose code expansion for op refs

### PR DESCRIPTION
**Pull request by tssweeney**

### CODE REVIEW

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewer.tsx [hunk 1]
```diff
@@ -354,10 +380,12 @@ export const ObjectViewer = ({apiRef, data, isExpanded}: ObjectViewerProps) => {
           const isTableRef =
             isRef(params.model.value) &&
             (parseRefMaybe(params.model.value) as any).weaveKind === 'table';
+          const isCode = params.model.isCode;
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewer.tsx:383]: Could be `const {isCode} = params.model;`

File: weave-js/src/components/PagePanelComponents/Home/Browse2/Browse2OpDefCode.tsx [hunk 2]
```diff
@@ -27,4 +31,19 @@ export const Browse2OpDefCode: FC<{uri: string}> = ({uri}) => {
...
     />
   );
+  if (maxRowsInView) {
+    let height = '100%';
+    const totalLines = text.result?.split('\n').length ?? 0;
+    const lineHeight = 18;
+    const padding = 20;
+    const maxHeight = maxRowsInView * lineHeight + padding;
+    const targetHeight = totalLines * lineHeight + padding;
+    if (targetHeight > maxHeight) {
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse2/Browse2OpDefCode.tsx:41]: How about:
```
    const showLines = Math.min(totalLines, maxRowsInView);
    const height = showLines * lineHeight + padding + 'px';
```

**Review by jamie-rasmussen — APPROVED** [review]
I'm seeing some sizing weirdness with the loading indicator in Browse2OpDefCode that we should address as a follow up.

File: weave-js/src/components/PagePanelComponents/Home/Browse2/Browse2OpDefCode.tsx (hunk 2 above)
Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse2/Browse2OpDefCode.tsx]: 👍

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewer.tsx (hunk 1 above)
Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewer.tsx]: 👍

**Pull request merged**